### PR TITLE
Feature/groups

### PR DESCRIPTION
**Pull request by andreas83**

### CODE REVIEW

File: app/controller/ScoreController.php [hunk 1]
```diff
@@ -13,7 +13,7 @@ class ScoreController
 {
     
     /**
-     * get like/dislike information for 
+     * get like/dislixxgddgke information for
```
Comment (chilimatic) [inline app/controller/ScoreController.php]: :D i see ...

File: app/model/Groups.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        
+        $stmt->execute();
+
+        
+    }
+    
+    
+    
+}
+?>
```
Comment (chilimatic) [inline app/model/Groups.php]: abschließenden php tags können zu error führen wenn irgendwer darunter noch ein \n einfügt. bei puren php files

File: gulpfile.js [hunk 3]
```diff
@@ -11,12 +11,11 @@ var gulp = require('gulp'),
     concat = require('gulp-concat'),
     cache = require('gulp-cache'),
     babel = require("gulp-babel"),
-    gutil = require('gulp-util'),
     sass = require('gulp-sass'),
     cssmin = require('gulp-cssmin'),
     del = require('del');
 
-
+/*    gutil = require('gulp-util'), */
```
Comment (chilimatic) [inline gulpfile.js:18]: comments ? srsly <3 the purpose would make more sense but you gotta start somewhere i guess :P

Comment (andreas83) [inline gulpfile.js:18]: i had some performance impacts and was trying to isolate the problem..  
anywayl look like i pushed a lot of crap on github last time. sry guys!

**Pull request closed without merging**